editor/plugins/gdextension/cpp_template/addon/library_name.gdextension

```diff
@@ -0,0 +1,21 @@
+[configuration]
+
+entry_symbol = "__LIBRARY_NAME___library_init"
+compatibility_minimum = __GODOT_VERSION__
```

Comment:
Also, a good opportunity to add `reloadable` entry: <https://github.com/godotengine/godot/pull/80284>
Disabled by default, I guess?

Reply:
Given that this is creating a project-specific GDExtension that the user will be actively working on, it may actually be appropriate to enable "reloadable" by default?